### PR TITLE
Finn

### PR DESCRIPTION
**Pull request by mcdfinn**
Merge

### CODE REVIEW

File: Smelter/word_whitelist.json [hunk 1]
```diff
@@ -1,5 +1,510 @@
 {
   "approved_words": [
+    "",
```
Comment (jyan-zarate) [inline Smelter/word_whitelist.json]: Why is this approved?

File: Smelter/word_whitelist.json [hunk 2]
```diff
@@ -1,5 +1,510 @@
...
+   "stone",
+   "an",
+   "door",
+   "root",
+   "lippitylippitynot",
+   "peter",
+   "on",
+   "up",
+   "along",
+   "rabbits",
```
Comment (jyan-zarate) [inline Smelter/word_whitelist.json:40]: shouldn't this be in the animal section?

File: Smelter/word_whitelist.json [hunk 3]
```diff
@@ -1,5 +1,510 @@
...
+   "locked",
+   "where",
+   "work",
+   "am",
+   "stone",
+   "an",
+   "door",
+   "root",
+   "lippitylippitynot",
+   "peter",
```
Comment (jyan-zarate) [inline Smelter/word_whitelist.json]: this is a name

File: Smelter/word_whitelist.json [hunk 4]
```diff
@@ -1,5 +1,510 @@
...
+   "ran",
+   "locked",
+   "where",
+   "work",
+   "am",
+   "stone",
+   "an",
+   "door",
+   "root",
+   "lippitylippitynot",
```
Comment (jyan-zarate) [inline Smelter/word_whitelist.json]: this isn't a word

File: Smelter/word_whitelist.json [hunk 5]
```diff
@@ -1,5 +1,510 @@
...
+   "peter",
+   "on",
+   "up",
+   "along",
+   "rabbits",
+   "jacket",
+   "still",
+   "cottontail",
+   "scratch",
+   "rabbit",
```
Comment (jyan-zarate) [inline Smelter/word_whitelist.json]: shouldn't this be in the animal section?

File: Smelter/word_whitelist.json [hunk 6]
```diff
@@ -1,5 +1,510 @@
...
+   "parsley",
+   "sparrows",
```
Comment (jyan-zarate) [inline Smelter/word_whitelist.json]: shouldn't this be in the animal section?

File: Smelter/word_whitelist.json [hunk 7]
```diff
@@ -1,5 +1,510 @@
...
+   "mischief",
+   "his",
+   "carrying",
+   "fast",
+   "twitched",
+   "been",
+   "back",
+   "going",
+   "thing",
+   "bunnies",
```
Comment (jyan-zarate) [inline Smelter/word_whitelist.json]: shouldn't this be in the animal section?

File: Smelter/word_whitelist.json [hunk 8]
```diff
@@ -1,5 +1,510 @@
...
+   "went",
+   "ate",
+   "most",
+   "through",
+   "came",
+   "round",
+   "sick",
+   "planting",
+   "quite",
+   "hoescrrritch",
```
Comment (jyan-zarate) [inline Smelter/word_whitelist.json]: This is not a word.

File: Smelter/word_whitelist.json [hunk 9]
```diff
@@ -1,5 +1,510 @@
...
+   "mother",
+   "thief",
+   "forgotten",
+   "foot",
+   "young",
+   "rushed",
+   "looking",
+   "hands",
+   "lived",
+   "firtree",
```
Comment (jyan-zarate) [inline Smelter/word_whitelist.json]: you mean "fir" and "tree"?

File: Smelter/word_whitelist.json [hunk 10]
```diff
@@ -1,5 +1,510 @@
...
+   "beautiful",
+   "big",
+   "bought",
+   "from",
+   "least",
+   "may",
+   "to",
+   "accident",
+   "he",
+   "cat",
```
Comment (jyan-zarate) [inline Smelter/word_whitelist.json:174]: shouldn't this be in the animal section?

File: Smelter/word_whitelist.json [hunk 11]
```diff
@@ -1,5 +1,510 @@
...
+   "somewhere",
+   "cry",
+   "wall",
+   "legs",
+   "some",
+   "answer",
+   "brown",
+   "straight",
+   "cousin",
+   "sneezedkertyschoo",
```
Comment (jyan-zarate) [inline Smelter/word_whitelist.json]: this isn't a word

File: Smelter/word_whitelist.json [hunk 12]
```diff
@@ -1,5 +1,510 @@
...
+   "losing",
+   "by",
+   "top",
+   "upsetting",
+   "gooseberry",
+   "bakers",
+   "thought",
+   "got",
+   "tried",
+   "bunny",
```
Comment (jyan-zarate) [inline Smelter/word_whitelist.json:250]: shouldn't this be in the animal section?

File: Smelter/word_whitelist.json [hunk 13]
```diff
@@ -1,5 +1,510 @@
...
+   "shoes",
+   "now",
+   "in",
+   "over",
+   "her",
+   "goldfish",
+   "plants",
+   "shoe",
+   "gave",
+   "faintingim",
```
Comment (jyan-zarate) [inline Smelter/word_whitelist.json]: isn't a word

File: Smelter/word_whitelist.json [hunk 14]
```diff
@@ -1,5 +1,510 @@
...
+   "fortunately",
+   "couldnt",
+   "ah",
+   "shall",
+   "help",
+   "trouble",
+   "alone",
+   "did",
+   "itdo",
```
Comment (jyan-zarate) [inline Smelter/word_whitelist.json]: isn't a word

File: Smelter/word_whitelist.json [hunk 15]
```diff
@@ -1,5 +1,510 @@
...
+   "nephew",
+   "dead",
+   "noble",
+   "precious",
+   "shouldnt",
```
Comment (jyan-zarate) [inline Smelter/word_whitelist.json:313]: should be "shouldn't"

File: Smelter/word_whitelist.json [hunk 16]
```diff
@@ -1,5 +1,510 @@
...
+   "myself",
+   "died",
+   "again",
+   "swear",
+   "its",
+   "disease",
+   "sort",
+   "imagine",
+   "proud",
+   "holmescoals",
```
Comment (jyan-zarate) [inline Smelter/word_whitelist.json]: isn't a word

File: Smelter/word_whitelist.json [hunk 17]
```diff
@@ -1,5 +1,510 @@
...
+   "study",
+   "word",
+   "yet",
+   "box",
+   "let",
+   "bad",
+   "hold",
+   "dying",
+   "sharp",
+   "asiatic",
```
Comment (jyan-zarate) [inline Smelter/word_whitelist.json:372]: pretty random word

File: Smelter/word_whitelist.json [hunk 18]
```diff
@@ -1,5 +1,510 @@
...
+   "cure",
+   "thateh",
```
Comment (jyan-zarate) [inline Smelter/word_whitelist.json:374]: isn't a word

File: Smelter/word_whitelist.json [hunk 19]
```diff
@@ -1,5 +1,510 @@
...
+   "yes",
+   "path",
+   "slop",
+   "here",
+   "fourth",
+   "fool",
+   "joke",
+   "appreciate",
+   "special",
+   "headi",
```
Comment (jyan-zarate) [inline Smelter/word_whitelist.json]: isn't a word

File: Smelter/word_whitelist.json [hunk 20]
```diff
@@ -1,5 +1,510 @@
...
+   "chance",
+   "friend",
+   "heart",
+   "ill",
+   "ivory",
+   "another",
+   "cramp",
+   "die",
+   "do",
+   "boxthis",
```
Comment (jyan-zarate) [inline Smelter/word_whitelist.json]: isn't a word

File: Smelter/word_whitelist.json [hunk 21]
```diff
@@ -1,5 +1,510 @@
...
+   "prove",
+   "certainly",
+   "this",
+   "is",
+   "will",
+   "sound",
+   "before",
+   "mind",
+   "youvery",
```
Comment (jyan-zarate) [inline Smelter/word_whitelist.json]: isn't a word

File: Smelter/word_whitelist.json [hunk 22]
```diff
@@ -1,5 +1,510 @@
...
+   "wednesday",
+   "are",
+   "matters",
+   "how",
+   "post",
+   "spreading",
+   "daya",
+   "breathing",
+   "londona",
+   "youre",
```
Comment (jyan-zarate) [inline Smelter/word_whitelist.json:469]: It's `you're` not `youre`

File: Smelter/word_whitelist.json [hunk 23]
```diff
@@ -1,5 +1,510 @@
...
+   "listen",
+   "done",
+   "be",
+   "effect",
+   "must",
+   "gone",
+   "victor",
+   "leave",
+   "reports",
+   "hidingplace",
```
Comment (jyan-zarate) [inline Smelter/word_whitelist.json]: you mean `hiding` and `place`?

File: Smelter/word_whitelist.json [hunk 24]
```diff
@@ -1,5 +1,510 @@
...
+   "fire",
+   "meive",
```
Comment (jyan-zarate) [inline Smelter/word_whitelist.json:486]: not a word

File: Smelter/word_whitelist.json [hunk 25]
```diff
@@ -141,9 +645,7 @@
     "authority",
     "available",
     "avoid",
-    "away",
```
Comment (jyan-zarate) [inline Smelter/word_whitelist.json:144]: why were these words removed?

File: Smelter/word_whitelist.json [hunk 26]
```diff
@@ -1139,44 +1614,29 @@
...
     "shortcut",
-    "use",
     "acivate",
-    "fight",
-    "dead",
-    "kill",
     "killed",
-    "doctor",
     "pharmacy",
-    "hospital",
```
Comment (jyan-zarate) [inline Smelter/word_whitelist.json:1170]: why were so many game related words removed?

File: word_scrambler.py [hunk 27]
```diff
@@ -1,9 +1,12 @@
...
+from colorama import Fore, Back, Style
 
+warning = """This script MUST be used with python3 to work properly. Instead of running python word_scrambler.py, run python3 word_scrambler.py"""
+print(Fore.RED + warning)
+print(Style.RESET_ALL)
 
-# Replace script w/ the text of what you're scrambling.
-script = """We the People of the United States, in Order to form a more perfect Union, establish Justice, insure domestic Tranquility, provide for the common defense, promote the general"""
-
+script = input("""Please enter the text you would like scrambled:""")
```
Comment (jyan-zarate) [inline word_scrambler.py:9]: this doesn't work with multiple lines

**Review by JCharante — CHANGES_REQUESTED** [review]
Was this even tested?

File: Smelter/word_whitelist.json [hunk 28]
```diff
@@ -305,7 +296,6 @@
    "plants",
    "shoe",
    "gave",
-   "faintingim",
    "fortunately",
    "couldnt",
```
Comment (jyan-zarate) [inline Smelter/word_whitelist.json:300]: `couldn't`

**Review by JCharante — APPROVED** [review]

**Pull request merged**